qbraid/runtime/origin/device.py

```
@@ -0,0 +1,99 @@
+    @property
+    def backend_name(self) -> str:
+        """Return the OriginQ backend name."""
+        return self._backend_name
+
+    def status(self) -> DeviceStatus:
+        """Return the current status of the OriginQ device."""
+        # OriginQ does not expose health endpoints; assume online when instantiated.
+        return DeviceStatus.ONLINE
+
+    def submit(
+        self, run_input: Union[QProg, list[QProg]], *, shots: int, **kwargs
```

Ruff (0.15.9)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)